README.md

```diff
@@ -26,18 +26,43 @@ If you believe you have found a security issue, please responsibly disclose by c
 4. PR should pass all the checks before merging.
 5. PR should be squashed and merged.
 
-### Common Commands
+
+## Modules
+
+This repository contains all components related to the CryoET backend. Below is a brief overview of each module:
+
+### api_server
+This module contains the v1 version of the GraphQL API for the CryoET backend. It is implemented using **Hasura**.
+**Note**: This version is on the path to deprecation. We should be using the v2 API for building new features.
+
+### apiv2
+This module contains the v2 version of the GraphQL API for the CryoET backend. It is implemented using **Platformics**, offering more features over v1.
+
+
+### docs
+This folder contains high-level documentation covering the overall architecture, and related processes.
+
+
+### ingestion_tools
+This module contains all the code related to the process of importing datasets into the backend, transforming the data into standardized format, validating the standardized data. It also contains the yamls that are used for configuring the ingestion process.
+
+
+### schema
+This module is responsible for maintaining the schemas used in both the ingestion configuration and the API. It is implemented using LinkML.
```

Comment:
can you add a note about `test_infra` and how it's used by apiv2 and ingestion tools tests?